LabExT/Movement/MoverNew.py

```diff
@@ -530,36 +533,49 @@ def move_absolute(
         if not movement_commands:
             return
 
-        path_planning = PathPlanning(chip)
+        with self.set_stages_coordinate_system(CoordinateSystem.CHIP):
+            path_planning = PathPlanning(chip)
```

Comment:
Changed the logic of `move_absolute`. A flag `with_lifted_stages` indicates whether the stages should be lifted/lowered beforehand. Path Planning knows nothing about this, it now simply reads the start Z-coordinate in the chip system and keeps it constant during the movement.